data/json/terrain.json

```diff
@@ -214,6 +224,7 @@
         "type" : "terrain",
         "id" : "t_ov_smreb_cage",
         "name": "small rebar roof cage",
+        "description": "A series of structural support crafted from rebar in order to allow the pouring of concrete for a more stable floor and roof.",
```

Comment:
Is this correct? `A series of structural support crafted from rebar`

Reply:
Yes, so instead of "A series of structural support `made from wood`" or material, it is of rebar.